src/Mod/PartDesign/fcsprocket/fcsprocketdialog.py

```diff
@@ -0,0 +1,66 @@
+# (c) 2020 Adam Spontarelli <adam@vector-space.org>
+#
+#   This program is free software; you can redistribute it and/or modify
+#   it under the terms of the GNU Lesser General Public License (LGPL)
+#   as published by the Free Software Foundation; either version 2 of
+#   the License, or (at your option) any later version.
+#   for detail see the LICENCE text file.
+#
+#   FCGear is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#   GNU Library General Public License for more details.
+#
+#   You should have received a copy of the GNU Library General Public
+#   License along with FCGear; if not, write to the Free Software
+#   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307
+
+from PyQt4 import QtGui as qt
```

Comment:
Do not use PyQt.

FreeCAD uses PySide. You should import from PySide.

Reply:
corrected in fdeef97